src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,5 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+    return boolean === true ? "Yes" : "No";
```

Comment:
Most of the work here uses 2 spaces for indentation, this should be kept the same across the whole project for consistency:
```
  return boolean === true ? "Yes" : "No";
```